src/Skybrud.Umbraco.BorgerDk/Caching/BorgerDkCachingService.cs

```diff
@@ -0,0 +1,121 @@
+﻿using Microsoft.Extensions.Caching.Distributed;
+using Microsoft.Extensions.Caching.Memory;
+using Microsoft.Extensions.Logging;
+using NPoco;
+using Skybrud.Umbraco.BorgerDk.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Umbraco.Cms.Core.Scoping;
+using Umbraco.Cms.Infrastructure.Persistence;
+using Umbraco.Extensions;
+
+namespace Skybrud.Umbraco.BorgerDk.Caching {
+    public class BorgerDkCachingService {
+        private readonly IMemoryCache cache;
+        private readonly IScopeProvider scopeProvider;
+        private readonly ILogger<BorgerDkCachingService> logger;
+        private readonly MemoryCacheEntryOptions cacheEntryOptions;
```

Comment:
Private fields should be prefixed with an underscore (see my comment about `.editorconfig`).